### PR TITLE
markdownlint-cli: Add default config file names and rule selection in args

### PR DESCRIPTION
**Pull request by berquist**
* The default configuration filenames are taken from the project readme.
* `--enable` and `--disable` allow specifying names of rules without putting them in a configuration file.

Please let me know if you'd like this tested.

### CODE REVIEW

**Review by jcs090218 — APPROVED** [review]
I've a quick glance, LGTM and CI passes. However, I don't have the push permission, so we will have to wait for other maintainers to review. ;)

**Review by cpitclaudel — APPROVED** [review]

**Comment by cpitclaudel** [comment]
Thanks!

**Pull request merged**
